cdap-ui/app/features/admin/controllers/namespace/datasets-ctrl.js

```diff
@@ -28,7 +28,12 @@ angular.module(PKG.name + '.feature.admin')
     myDatasetApi.list(params)
       .$promise
       .then(function (res) {
-        $scope.dataList = $scope.dataList.concat(res);
+        if (!Array.isArray(res)) {
+          return;
+        }
+        $scope.dataList = res.filter(function(dataset) {
+          return dataset.type !== 'externalDataset';
+        });
```

Comment:
Need to be concatenated to the dataList array, because the Streams callback also concat to the same array